### PR TITLE
Fixes #42 .Update contact.md

### LINKED ISSUE #42: Add more social links on contact page
Add more social links on contact page

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Fixes issue #42 
Added more social profiles in contact page

### CODE REVIEW

**Comment by arjunpandey2003** [comment]
https://arjunpandey2003.github.io/2014.fossasia.org/

File: contact.md [hunk 1]
```diff
@@ -15,6 +15,16 @@ Google+: [https://plus.google.com/u/0/108920596016838318216](https://plus.google
...
 
+Instagram: [https://www.instagram.com/fossasia/?hl=en](https://www.instagram.com/fossasia/?hl=en)
+
+Twitter: [https://twitter.com/fossasia](https://twitter.com/fossasia)
+
+Youtube: [https://www.youtube.com/channel/fossasia](https://www.youtube.com/channel/UCQprMsG-raCIMlBudm20iLQ)
+
+Contribute at: 
+
+Github:[https://github.com/fossasia/fossasia.org](https://github.com/fossasia/fossasia.org)
```
Comment (the-codingninja) [inline contact.md]: Can you please make this link go to `fossasia` instead of `fossasia.org`

**Review by the-codingninja — CHANGES_REQUESTED** [review]
Can you make suggested changes.
Rest LGTM!

**Comment by arjunpandey2003** [comment]
@ms10398 done

**Review by ParthS007 — APPROVED** [review]

**Comment by the-codingninja** [comment]
Please change the link to `https://github.com/fossasia`

**Comment by arjunpandey2003** [comment]
@ParthS007 can you approve and merge thanks

**Review by the-codingninja — APPROVED** [review]

**Review by dilpreetsio — APPROVED** [review]

**Pull request merged**